mkdocs.yml

```diff
@@ -320,6 +320,7 @@ nav:
             - Configure checkout: commerce/checkout/configure_checkout.md
             - Customize checkout: commerce/checkout/customize_checkout.md
             - Checkout API: commerce/checkout/checkout_api.md
+            - Extend checkout: commerce/checkout/extend_checkout.md
```

Comment:
We now have one page for extending and one for customizing checkout. [Here](https://doc.ibexa.co/en/latest/commerce/commerce/) customization is added under extending commerce header. New page should be also added to this landing page.
In my opinion, we need a different name.
Please move it above API.